services/m3aggregator/config/aggregator.go

```diff
@@ -281,40 +285,70 @@ func (c *streamConfiguration) NewStreamOptions(instrumentOpts instrument.Options
 	return opts, nil
 }
 
-// flushConfiguration contains configuration for flushing metrics.
-type flushConfiguration struct {
-	// Flushing function type.
+// flushHandlerConfiguration contains configuration for flushing metrics.
+type flushHandlerConfiguration struct {
+	// Flushing handler type.
 	Type string `yaml:"type" validate:"regexp=(^blackhole$|^logging$|^forward$)"`
```

Comment:
This config seems weird, why restrict the handler types in both a const as well as yaml validation. Are we going to chain these handlers? Do we need all handlers or are any of them optional? Any reason why we shouldn't just define/build a handler chain in code as we need them? 

Also, the forward only applies to the Forward Handler - this seems odd too.